compiler/luci/import/include/luci/Import/GraphBuilderRegistry.h

```diff
@@ -73,11 +90,17 @@ class GraphBuilderRegistry final : public GraphBuilderSource
     _builder_map[op] = std::move(builder);
   }
 
+  void add(std::unique_ptr<TensorBuilder> &&builder)
+  {
+    _tensor_builders.at(uint32_t(builder->builder_type())) = std::move(builder);
+  }
+
 private:
   const GraphBuilderSource *_parent = nullptr;
 
 private:
   std::map<const circle::BuiltinOperator, std::unique_ptr<GraphBuilderBase>> _builder_map;
+  std::vector<std::unique_ptr<TensorBuilder>> _tensor_builders;
```

Comment:
nitpick: you can use 
``` c++
std::array<std::unique_ptr<TensorBuilder>, TensorBuilderType::COUNT> _tensor_builder = {};
```
 or
 ``` c++
 std::unique_ptr<TensorBuilder> _tensor_builder[TensorBuilderType::COUNT] = {}
 ```

Reply:
Personally, I prefer first option. Done!